### PR TITLE
Sponsors per row

### PR DESCRIPTION
**Pull request by tepkenvannkorn**
Fix in 768px and larger screens:
Siam - 2 sponsors per row
Silom - 3 sponsors per row
Asok - 4 sponsors per row
Ari - 5 sponsors per row
Nana - 6 sponsors per row

### CODE REVIEW

**Comment by drebbits** [comment]
@tepkenvannkorn is this in reference to the changes the Sponsors team is requesting:

> We need change the number of sponsors displayed in one line of each plan.
> - asok 3
> - ali (ari) 4
> - nana 5
> And We want to change the sponsor’s rectangular logo to the center, referring to the WordCamp US sponsor list.

File: css/sass/_sponsor.scss [hunk 1]
```diff
@@ -134,12 +129,11 @@
...
             display: flex;
             flex-flow: row wrap;
-
             .wcorg-sponsor {
                 width: calc(100%/3);
                 margin-left: 0;
-                height: auto;;
-
+                height: auto;
+                ;
```
Comment (drebbits) [inline css/sass/_sponsor.scss]: @tepkenvannkorn this might unintentionally added semi-colon. Can you check?

Comment (tepkenvannkorn) [reply css/sass/_sponsor.scss]: removed now.

**Comment by drebbits** [comment]
@tepkenvannkorn 

Here's the confirmed number of sponsors per row per level:

Siam and Silom: 2
Asok: 3
Ari: 4
Nana: 5

As noted earlier in the thread here, they are also exploring the idea of having a background to the sponsors logo and boxed in a rectangular container (similar to WCUS 2019 - https://2019.us.wordcamp.org/sponsors/)

**Pull request closed without merging**